src/main/java/io/fabric8/jenkins/openshiftsync/JobProcessor.java

```diff
@@ -137,8 +137,15 @@ private void populateNamespaceFolder(Jenkins activeInstance, ItemGroup parent, S
 					activeInstance.createProjectFromXML(jobName, jobStream).save();
 				}
 
-				logger.info("Created job " + jobName + " from BuildConfig " + NamespaceName.create(buildConfig)
-						+ " with revision: " + buildConfig.getMetadata().getResourceVersion());
+				String autostart = getAnnotation(buildConfig, AUTOSTART);
+				if (Boolean.parseBoolean(autostart)) {
+					logger.info("Created and automatically starting job " + jobName + " from BuildConfig " 
+					        + NamespaceName.create(buildConfig)	+ " with revision: " + buildConfig.getMetadata().getResourceVersion());
```

Comment:
This part needs to be done in all cases. It must be extracted from if/else.

Reply:
There's a log message in both cases.
one '"Created and automatically starting job' line 140 the other 'Created job' line 147

Reply:
should the original message be printed in all cases an additional for this feature?

Reply:
yes, exactely, let's print the original message in all cases; and add a specific one for the autostart.